### DISCUSSION TITLE
EPEVER ID

### DISCUSSION THREAD

**Discussion by HauWii** [softwarecrash/EPEver2MQTT#40] [General]
Hallo,

ich wollte einmal nachfragen ob es für mein aktuelles Problem eine Lösung gibt. Ich habe 4 EPEVER TRACER 2206 AN mit dem ESP8266 und EPEver2MQTT. Den ESP8266 habe ich ins Gerhäuse integriert (also jeder Regler hat einen eigenen ESP8266), dazu habe ich eine Stiftleiste eingelötet, hole mir an einem Pad die +5V und musste eine Leitung durchtrennen. Ich arbeite noch an einer Lösung um per Optokoppler die Kommunikation über das VD1500 IC wieder herzustellen wenn der ESP8266 nicht mehr kommuniziert...Also ein Hin und Herschalten damit man mit der Solar Station MonitorV1.95 im Notfall noch parametrieren kann...
Gerade habe ich leider einen Regler an den ich nicht dran komme um die ID zu ändern. Diese ist hier 4 (bei den anderen sonst immer 1). Wenn ich die Quantity auf 4 stelle komme ich zwar über MQTT EP_4 an Daten, jedoch auf der Website sehe ich nur Device 1 no Data und man kann nicht weiterblättern.
Was kann ich da tun?

Vielen Dank für eure Mühen...

**Comment by softwarecrash** [comment]
hi,
Tatsächlich ist das WebUI aktuell nicht dafür ausgelegt um auf ID 4 daten zu haben wenn die davor nie daten geliefert haben, ggf kann ich die bläter funktion dahingehend noch anpassen. aber das ist nichts was ich grad auf die schnelle hinbekomme.

hast du mal versucht den von ID 4 auf ID 1 umzustellen?
geht übrigens auch mit der firmware
https://github.com/softwarecrash/EPEver2MQTT/wiki/Multi-Device-Tutorial

**Reply by HauWii** [reply]
Wunderbar, genau das hab ich gesucht und bisher noch nicht gefunden... Und es hat auch geklappt! Besten Dank

Ich hab gerade noch ein anderes Anliegen bzgl. Schalten des LOAD_STATE über MQTT gepostet. Könntest du mir dazu auch noch einen Tipp geben?
